vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/TimelineEventVisibilityHelper.kt

```diff
@@ -117,7 +117,7 @@ class TimelineEventVisibilityHelper @Inject constructor(private val userPreferen
             rootThreadEventId: String?
     ): Boolean {
         // If show hidden events is true we should always display something
-        if (userPreferencesProvider.shouldShowHiddenEvents()) {
+        if (userPreferencesProvider.shouldShowHiddenEvents() && !isFromThreadTimeline) {
```

Comment:
Does it means that a message edition in a thread will not be displayed as a debug event in the thread? This is fine, just want to confirm. I will do some test when the APK will be ready

Reply:
Exactly! It will be displayed in the main timeline. We can of course further improve all those thing when threads are more stable